examples/bri-3/src/bri/zeroKnowledgeProof/services/blockchain/ethereum/ethereum.service.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from 'hardhat';
+import { readFile, writeFile } from 'fs/promises';
+import { Injectable } from '@nestjs/common';
+import { IBlockchainService } from '../blockchain.interface';
+import { Contract } from 'ethers';
+
+@Injectable()
+export class EthereumService implements IBlockchainService {
+  public async deployContract(contractName: string): Promise<void> {
```

Comment:
we need to discuss who should be doing this deployment and when. mil5 stuff, i ll open an issue.

Reply:
Done. #751